src/main/java/com/sap/sgs/phosphor/fosstars/data/github/NvdEntryMatcher.java

```diff
@@ -156,21 +174,111 @@ private boolean match(VendorData vendorData) {
   }
 
   /**
-   * Checks if product data from an NVD entry matches to the project's owner and name.
+   * Checks if product data from an NVD entry matches to the project's name.
    *
    * @param productData The produce data.
-   * @return True if the product data matches to the project's owner and name, false otherwise.
+   * @return True if the product data matches to the project's name, false otherwise.
    */
   private boolean match(ProductData productData) {
     return match(project.name(), productData.getProductName());
   }
 
   /**
-   * Returns true if one of two strings includes the other one.
+   * Checks if the a string is similar to another. It goes through a series of controlled checks to
+   * confirm this.
+   * 
+   * @param String one.
+   * @param String two.
+   * @return True if the accumulated points is greater than the threshold. Otherwise False.
+   */
+  private boolean match(String one, String two) {
+    if (one.equalsIgnoreCase(two)) {
+      return true;
+    }
+
+    int point = 0;
+
+    if (jaroWinklerSimilarityCheck(one, two)) {
+      point += 1;
+    }
+
+    if (longestCommonSubsequenceCheck(one, two)) {
+      point += 1;
+    }
+
+    return point > 0 && point / 2 > METRIC_THRESHOLD;
```

Comment:
`point / 2` results to an integer, not a double. If `point` is 1, then the result will always be 0. If you'd like to use a double threshold here, you need to do `(double) point / 2`. Although I am not sure if we need a threshold here if we have only two checks. The current check may be replaced with `jaroWinklerSimilarityCheck(one, two) && longestCommonSubsequenceCheck(one, two)`.

Reply:
We also have a bug in `UsesGithubForDevelopment` that also devides two integers

https://github.com/SAP/fosstars-rating-core/blob/master/src/main/java/com/sap/sgs/phosphor/fosstars/data/github/UsesGithubForDevelopment.java#L111